ext/page_log/palm/palm.c

```diff
@@ -526,7 +527,7 @@ palm_handle_get(WT_PAGE_LOG_HANDLE *plh, WT_SESSION *session, uint64_t page_id,
     palm = palm_handle->palm;
     palm_delay(palm);
 
-    VERBOSE_PRINT(palm_handle->palm, "palm_handle_get\n");
+    PALM_VERBOSE_PRINT(palm_handle->palm, "palm_handle_get\n");
```

Comment:
Optional change: Maybe include the page ID here, so that the beginning and end print statements can be matched if we're multithreaded? I'm not sure it's necessary, but I sometimes find that helpful...